releases/cloud/point-2024-09-12-release-note.md

```diff
@@ -0,0 +1,42 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.4.61.2
+---
+
+# September 12, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+## Improvements & Bug Fixes
+
+* **Improvements made to reports.**
+
+  * **The Sites Overview report and Teams & Groups Overview report** now have an additional column available to select from the column chooser: Last Reviewed.  
```

Comment:
add info that this columns shows the date and time of the last Access Review performed on the workspace